Add required sepolicy for virtio

renchenglei opened the pull request:
Add the required sepolicy rules for virtio. Virtio can boot
to UI with selinux in enforcing mode.

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1438

renchenglei committed:
Add required sepolicy for virgl

Add the required sepolicy rules for virgl. Virgl can boot
to UI with selinux in enforcing mode.

Tracked-On: OAM-88699
Signed-off-by: Ren, Chenglei <chenglei.ren@intel.com>

renchenglei commented:
@YuanjunHuang, please help take a look, thanks a lot!

renchenglei commented:
@YuanjunHuang, please help take a look, thanks a lot!

renchenglei commented:
@feijiang1, please help take a look, thanks a lot!

feijiang1 commented on graphics/mesa/file_contexts:
```diff
@@ -22,9 +22,12 @@
 /(vendor|system/vendor)/lib(64)?/libglapi\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/i965_dri\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/virtio_gpu_dri\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/gallium_dri\.so u:object_r:same_process_hal_file:s0
```
I think we haven't used gallium_dri.so

renchenglei replied:
When we enable virtio, the gallium_dri.so is built out. Maybe gallium_dri.so is one dependent library for virtio?

feijiang1 replied:
for virtio gpu implementation, we need virtio_gpu_dri.so. Gallium_dri.so is sw opengl implementation. In Celadon, currently we use swiftshader instead of gallium_dri.so.

feijiang1 replied:
my mistake, gallium_dri.so is not for software opengl implementation, it is necessary to work with virtio_gpu_dri.so

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1273

renchenglei commented:
@kwang13, could you help take a review for this changes? Thanks a lot!

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1274

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1275

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1276

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1277

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/647

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Ren, Chenglei" <chenglei.ren@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/1344
